example_package/README.md

```diff
@@ -0,0 +1,454 @@
+<!-- Jak nie umiesz otworzyć by ładniej wyglądało, tu jest czytelniej -->
+<!-- https://github.com/Stowarzyszenie-Talent/st-make/tree/main/example_package -->
+
+# Spis Treści
+
+- [Szablon paczki](#szablon-paczki)
+- [Ogólne informacje](#ogólne-informacje)
+- [doc](#doc)
+  - [talentTex.cls](#talenttexcls)
+- [prog](#prog)
+  - [Rozwiązania](#rozwiązania)
+  - [ingen](#ingen)
+  - [inwer](#inwer)
+  - [checkerka](#checkerka)
+  - [oi.h](#oih)
+    - [Scanner (Wczytywanie)](#scanner-wczytywanie)
+    - [CheckerVerdict](#checkerverdict)
+    - [checker_test](#checker_test)
+    - [InwerVerdict](#inwerverdict)
+    - [bug](#bug)
+    - [oi_assert](#oi_assert)
+    - [Random](#random)
+- [config.yml](#configyml)
+  - [Interactive tasks](#interactive-tasks)
+  - [Time](#time)
+  - [Memory](#memory)
+  - [Title](#title)
+  - [Scores](#scores)
+  - [Task ID](#task-id)
+  - [Contest type](#contest-type)
+  - [Expected scores](#expected-scores)
+- [in i out](#in-i-out)
+- [dlazaw](#dlazaw)
+
+# **Szablon paczki**
+
+Jest to przykładowy szablon paczki, który zalecamy używać.
+Jedynie dla zadań interaktywnych jest on inny.
+(Jeszcze nie zrobiliśmy dla niego szablonu).
+W przypadku chęci zrobienia takiego zadania obecnie najlepiej skontaktuj się z kimś kto wie jak to się robi.
+
+Aktualna wersja paczki zawsze znajduje się zawsze
+[na githubie](https://github.com/Stowarzyszenie-Talent/st-make/tree/main/example_package).
+`st-make init`, pobiera właśnie tą wersję.
+
+# **Ogólne informacje**
+
+1. Do kompilacji paczki używamy skryptu `st-make`.
+Jest on dostępny [na githubie](https://github.com/Stowarzyszenie-Talent/st-make).
+1. Każde zadanie ma `tag` - jest to najczęściej 3 literowy skrót.
+Wszystkie nazwy plików w paczce zaczynamy od niego.
+W przykładowej paczce jest to `abc`.
+
+# **doc**
+
+Ten folder zawiera wszystkie pliki tekstowe (pdf, tex, doc, img, ...).
+
+- `{TAG}zad.tex` - treść zadania, (przyjmowane jest też w wordzie).
+- `{TAG}opr.tex` - dokument z opracowaniem zadania.
+Posiada wszelkie informacje techniczne o zadaniu.
+- `{TAG}opi.tex` - dokument z opisem rozwiązania.
+
+Do kompilacji użyj `st-make doc`.
+
+## **talentTex.cls**
+
+Jest to nasza klasa używana w plikach `.tex`.
+Nadaje ona odpowiedni wygląd dokumentą i udostępnia kilka ułatwień w pisaniu.
+Automatycznie tworzy ona nagłówki i stopki.
+Wystarczy że stworzymy treść pomiędzy znacznikami `\start` i `\finish`.
+Dodatkowo udostępnia następujące rzeczy:
+
+- `\tc{n}` - koloruje na talentowy kolor "n".
```

Comment:
koloruje podany tekst. O ile nie ma jeszcze lepszego sposobu, podać też mój brzydki chack na indeksy dolne

Reply:
to możesz go implementować, ja dodałem swoje obejście używające tylko w trybie matematycznym gdzie długość wyrażenia jest dłuższa od 1 innej czcionki.